src/dotnet/commands/dotnet-restore3/Program.cs

```diff
@@ -97,6 +103,13 @@ public static int Run(string[] args)
                     msbuildArgs.Add($"/p:RestoreIgnoreFailedSources=true");
                 }
 
+                // By default restore is recursive, if set to false turn it off.
```

Comment:
'--no-dependencies' is the name of the option for build and build3. This should match